cli/pcluster/commands.py

```diff
@@ -111,10 +112,24 @@ def _check_for_updates(pcluster_config):
         utils.check_if_latest_version()
 
 
+def _validate_cluster_name(cluster_name):
+    if not re.match(PCLUSTER_NAME_REGEX % (PCLUSTER_NAME_MAX_LENGTH - 1), cluster_name):
+        LOGGER.critical(
```

Comment:
I'd not log this as critical, but rather as error. Also you could simply use the error function from utils.

As a side node: in general favor format over printf based formatting (%)- https://realpython.com/python-string-formatting/
With logger you don't need to use `%` but just pass the variables as arguments after the parametrized string

Reply:
used LOGGER.error instead, thank you for the link